src/app/containers/ConsentBanner/Canonical/logic/setCookieOven.test.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable global-require */
```

Comment:
It's great to have this test coverage, covering cases for each of the loggers (console and defined externally).